Week2/js-exercises/ex2-whatsYourMondayWorth.js

```diff
@@ -14,25 +14,51 @@
 
 function dayWorth(tasks, hourlyRate) {
   // put your code in here, the function does returns a euro formatted string
+  const amountMoney = tasks
+    .map(number => number.duration / 60 * hourlyRate)
```

Comment:
Use better naming. in this case `tasks.map(task => task.duration...`
each of object in the mondayTasks variable is a task so it is better be called `task` than `number`